packages/worker/src/worker-options.ts

```diff
@@ -256,6 +256,21 @@ export interface WorkerOptions {
    */
   maxConcurrentWorkflowTaskExecutions?: number;
 
+  /**
+   * `maxConcurrentWorkflowTaskPolls` * this number = the number of max pollers that will
+   * be allowed for the nonsticky queue when sticky tasks are enabled. If both defaults are used,
+   * the sticky queue will allow 8 max pollers while the nonsticky queue will allow 2. The
+   * minimum for either poller is 1, so if `maxConcurrentWorkflowTaskPolls` is 1 and sticky queues are
+   * enabled, there will be 2 concurrent polls.
+   *
+   * ⚠️T This API is experimental and may be removed in the future if the poll scaling algorithm changes.
+   *
+   * @experimental
+   *
+   * @default 0.2
+   */
+  nonStickyToStickyPollRatio?: number;
```

Comment:
This must also be added to [core-bridge's `WorkerOptions`](https://github.com/bergundy/sdk-typescript/blob/pr/bergundy/1254/packages/core-bridge/ts/index.ts#L264), since we're reading it from Rust side.

Reply:
Have to admit though that, as it currently stands, this is somewhat maintenance overhead with little gain, as all props from the "outer" WorkerOptions are passed to bridge's createWorker anyway 🤔.

Could be pertinent to consider making a new object [here](https://github.com/bergundy/sdk-typescript/blob/pr/bergundy/1254/packages/worker/src/worker.ts#L221), matching exactly the bridge's expectation...

Reply:
I'll add it for the typing and obviously the right thing to do. FYI I tested and it's not required.